resources/views/__dashboard.blade.php

```diff
@@ -0,0 +1,18 @@
+<h1 class="mb-3 font-bold">Dashboard</h1>
+
+<div class="flex">
+    <div class="flex-1">
+        <img 
+            src="/image/sample-avatar.jpeg" 
+            alt="avatar"
+        >
+    </div>
+
+    <div class="flex-1 list-group p-3">
+            <p>
+                {{ Auth::user()->name }}
+            </p>
+            <a class="underline" href="#">Learned 20 Words</a>
+            <a class="underline" href="#">Learned 5 Lessons</a>
+    </div>
+</div>
```

Comment:
if ka bantay ka aning naay murag no entry sign, meaning always dapat naa kay space at the end of the file.

Reply:
I'll check the PR every time mo create ko new sir, kay di nako ma kita ang no entry sign sa VScode, and will commit the changes. (bow)